components/identity-mgt-new/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/IdentityMgtConfig.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.identity.mgt;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.mgt.constants.IdentityMgtConstants;
+import org.wso2.carbon.identity.mgt.mail.DefaultEmailSendingModule;
+import org.wso2.carbon.identity.mgt.password.DefaultPasswordGenerator;
+import org.wso2.carbon.identity.mgt.password.RandomPasswordGenerator;
+import org.wso2.carbon.identity.mgt.policy.PolicyRegistry;
+import org.wso2.carbon.identity.mgt.services.IdentityMgtService;
+import org.wso2.carbon.identity.mgt.services.impl.IdentityMgtServiceImpl;
+import org.wso2.carbon.identity.mgt.store.UserIdentityDataStore;
+import org.wso2.carbon.user.api.RealmConfiguration;
+import org.wso2.carbon.user.core.jdbc.JDBCRealmConstants;
+import org.wso2.carbon.utils.CarbonUtils;
+import org.wso2.carbon.utils.multitenancy.MultitenantConstants;
+
+import java.util.*;
```

Comment:
Do not import *. Import only the required classes one by one in java.util package